plugin/normalize.go

```diff
@@ -108,11 +108,16 @@ func SplitHostPort(s string) (hosts []string, port string, err error) {
 	}
 
 	// Check if it parses as a reverse zone, if so we use that. Must be fully specified IP and mask.
-	_, n, err := net.ParseCIDR(s)
+	ip, n, err := net.ParseCIDR(s)
 	if err != nil {
 		return []string{s}, port, nil
 	}
-
+	if ip.To4() == nil { // v6 address, if the mask if not on a octet, it's not a valid cidr (we don't need to split v6 like v4)
```

Comment:
Is this true? I think its typical that ipv6 prefix fall on boundaries, but I didnt think it was required.